gallery/workbook/microsoft.insights-components.json

```diff
@@ -178,50 +178,44 @@
                     "id": "Workbooks/Product Analytics using the Click Analytics Plugin/HEART Analytics - All Sections",
                     "name": "HEART Analytics - All Sections",
```

Comment:
has anyone complained that these are really long names in gallery? 

they all show up as "HEART Analytics Section- ..." 

![image](https://user-images.githubusercontent.com/10158007/142315748-a969cdb4-a82d-4711-84d1-5efd28b38ea3.png)

should that be at the end?

like "Heart Analytics - Overview",  "Adoption - HEART Analytics", "Engagement - HEART Analytics", etc, so the distinguishing thing is *first*, not at the end hidden in the overflow?